packages/application-shell/src/components/quick-access/quick-access.js

```diff
@@ -283,5 +327,36 @@ export default compose(
     'canViewDashboard',
     'canViewDiscounts',
     'customApplications',
-  ])
+  ]),
+  connect(
+    null,
+    (dispatch, props) => ({
+      pimSearchProductIds: searchText =>
```

Comment:
This is where we query pim-search using full-text search.

Reply:
I want to keep them close to the code and don't see an upside in that. In the beginning I even had

```
connect(
  null,
  (dispatch) => ({ dispatch })
)
```

which allowed me to "inline" the `sdkActions.post` call to where it's actually being triggered. I liked that as it kept things together. I ultimately moved it down with this expression on my face 😐 as it's "the way it's supposed to be" and because it felt kinda weird to just pass `dispatch`.